04-Multitasking/asm.h

```diff
@@ -2,6 +2,7 @@
 #define __ASM_H_
 
 unsigned int *activate(unsigned int *stack);
+void init_activate_env(unsigned int *stack);
```

Comment:
`init_activate_env` might be a misleading name since it is specific to two operations: init and activate. However, its functionality is to construct task-switching environment.

Reply:
But we use activate to switch task; I think it means init the environment that activate can function.
(It's copied from 06-Preemptive, I'm not pretty sure about the naming issue.)
Or is any suggestion for the function name?